src/user/oci/.micronaut/test-resources/test-resources-port.txt

```diff
@@ -0,0 +1 @@
+50664
```

Comment:
Remove and add `.micronaut/` to the root `.gitignore` file

Reply:
ok done